data/styles/vehicle/include/priorities_2_BG-top.prio.txt

```diff
@@ -20,8 +20,11 @@ landuse-reservoir                                   # area z12- (also has captio
 landuse-salt_pond                                   # area z1-
 natural-water                                       # area z1- (also has caption z10-)
 natural-water-basin                                 # area z1- (also has caption z10-)
+natural-water-ditch                                 # area z1- (also has caption z10-)
+natural-water-drain
```

Comment:
its good to include automatic comments changes into the main commit also (not into "regenerate");
the rule of thumb is - if there were manual changes to the priority files (in this case - new types added) then include into the main commit, if prio files changes were purely automatic (e.g. type's visibility changed in *.mapcss) then include them into the "regenerate" commit

I know its a bit of a fine line :)

Reply:
ah ok, will keep in mind for the future :)